toolchain.cmake

```diff
@@ -56,12 +56,13 @@ set(MICROCHIP_MCU "${MICROCHIP_MCU}"
     CACHE STRING "full model number of the target Microchip MCU"
 )
 
-
 # known 8-bit MCU families
 list(APPEND MICROCHIP_FAMILIES_8
     PIC12F
     PIC16F
     PIC18F
+    ATtiny
+    ATxmega
```

Comment:
What about ATmega parts?

Reply:
I do not a testing hardware for ATmega. I can add it, but untested.